astropy/io/votable/tests/table_test.py

```diff
@@ -286,5 +287,6 @@ def test_conf_pedantic_true(self, tmpdir):
 
             reload_config('astropy.io.votable')
 
-            with pytest.raises(VOWarning):
-                parse(get_pkg_data_filename('data/gemini.xml'))
+            with pytest.raises(AstropyDeprecationWarning):
+                with pytest.raises(VOWarning):
+                    parse(get_pkg_data_filename('data/gemini.xml'))
```

Comment:
I think this works too.
```suggestion
            with pytest.raises(AstropyDeprecationWarning), pytest.raises(VOWarning):
                parse(get_pkg_data_filename('data/gemini.xml'))
```

Reply:
This would work, but it would also exceed the 80 character line length limit.

Reply:
Our limit is 100 😸 

https://github.com/astropy/astropy/blob/46cfe3fc234c896e71a795c891642544ae676ded/setup.cfg#L153

Reply:
I pushed the second commit before seeing your answer, and I don't think the nested `with` statements here are bad enough to justify making another commit just to replace them, but I can change them if anyone requests more changes or if I'm asked to squash the commits.